ppr-ui/src/views/amendment/AmendRegistration.vue

```diff
@@ -195,11 +214,16 @@ export default class AmendRegistration extends Vue {
   }
 
   private confirmAmendment (): void {
-    this.$router.push({
-      name: RouteNames.CONFIRM_DISCHARGE,
-      query: { 'reg-num': this.registrationNumber }
-    })
-    this.emitHaveData(false)
+    if (this.debtorValid && this.securedPartiesValid && this.registrationLengthTrustValid && this.collateralValid) {
+      // TODO: change route name
+      this.$router.push({
+        name: RouteNames.CONFIRM_DISCHARGE,
```

Comment:
change route name or the route isn't there yet? When we comment for something to be done in a future ticket we use FUTURE instead of TODO fyi (so that in the PR I know you didn't write a comment for yourself + forget)

Reply:
created a route name for now